src/store/quotes.js

```diff
@@ -0,0 +1,33 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const quotes = createSlice({
+  name: 'quotes',
+  initialState: {
+    availableQuotes: [],
+    quoteIndex: 0
+  },
+  reducers: {
+    nextQuote(state) {
+      const hi = state.availableQuotes.length - 1;
```

Comment:
Hi! 

I would put this inline. Is it really - 1?

Reply:
Yes, i've added a tiny bit of documentation to explain the slice a bit further